features/bigint64array.yml

```
@@ -0,0 +1,14 @@
+name: BigInt64Array
+description: "The `BigInt64Array` and `BigUint64Array` typed arrays represent 64-bit signed/unsigned integers in the platform byte order."
```

```suggestion
description: "The `BigInt64Array` and `BigUint64Array` typed arrays represent 64-bit integrers, respectively signed and unsigned, in the platform byte order."
```

Not sure if the sentence is more parsable this way, but the `signed/unsigned` confused me for a bit.

I'm not familiar with what the "platform byte order" is. Is it needed here? Is there another way to convey the same meaning without having to write a super long explanation? Or is it fine because people who reach for BigInt64Arrays will know what this is?

---

We can also leave out platform byte order. I think it may not be that important. I basically stole this from https://github.com/web-platform-dx/web-features/pull/1530, so if we remove it here and we should remove it there as well.